Trench Warfare/map.xml

```diff
@@ -0,0 +1,187 @@
+<?xml version="1.0"?>
+<map proto="1.4.0">
+<name>Trench Warfare</name>
+<version>0.1</version>
+<objective>Be the team with the least amount of deaths after 10 minutes.</objective>
+<authors>
+    <author uuid="1379cb6e-f291-4498-9807-e636f9674ac0"/> <!-- SH4D0W_HAWK -->
+</authors>
+<contributors>
+    <contributor uuid="6968cbde-c2b7-479c-a8da-f33f109f013b" contribution="XML"/> <!-- Brottweiler -->
+</contributors>
+<score>
+    <kills>1</kills>
+    <deaths>1</deaths>
+</score>
+<time>10m</time>
+<rules>
+    <rule>Switching/exchanging classses or items within a class is illegal.</rule>
+</rules>
+<hunger>
+    <depletion>off</depletion>
+</hunger>
+<timelock>on</timelock>
+<teams>
+    <team id="blues" color="blue" max="20" show-name-tags="allies">Blue</team>
+    <team id="reds" color="dark red" max="20" show-name-tags="allies">Red</team>
+</teams>
+<spawns> <!-- Not changed yet -->
+    <spawn team="blues">
+        <regions yaw="0">
+            <cylinder base="0.5, 20, -70.5" radius="2.5" height="1"/>
+        </regions>
+    </spawn>
+    <spawn team="reds">
+        <regions yaw="180">
+            <cylinder base="0.5, 20, 70.5" radius="2.5" height="1"/>
+        </regions>
+    </spawn>
+    <default>
+        <regions yaw="0">
+            <cylinder base="0.5, 8, 0.5" radius="10.5" height="1"/>
+        </regions>
+    </default>
+</spawns>
+<classes family="offensive">
+    <class name="Soldier" default="true" description="Basic infantry equipped with medium armor and weapon." icon="iron sword">
+        <kit>
+            <item slot="0">iron sword</item>
+            <item slot="1" amount="16">egg</item>
+            <item slot="2" amount="5">grilled pork</item>
+            <helmet material="diamond helmet"/>
+            <chestplate material="iron chestplate"/>
+            <leggings material="iron leggings"/>
+            <boots material="iron boots"/>
+        </kit>
+    </class>
+</classes>
+<filters>
+    <filter id="only-red">
+        <team>red</team>
+    </filter>
+    <filter id="only-blue">
+        <team>blue</team>
+    </filter>
+    <filter id="no-tnt">
+        <deny>
+            <block>tnt</block>
+        </deny>
+    </filter>
+    <filter id="break-restrictions"> <!--Deny breaking stone brick and iron objects-->
+        <deny>
+            <block>iron block</block>
+            <block damage="5">step</block>
+            <block>redstone wire</block>
+            <block>wall sign</block>
+            <block>stone plate</block>
+            <block>smooth brick</block>
+            <block>iron fence</block>
+            <block>smooth stairs</block>
+            <block>wood button</block>
+        </deny>
+    </filter>
+</filters>
+<regions>
+    <!--Playing Field Regions-->
+    <union name="bases">
```

Comment:
In proto 1.4.0, the `name` attribute in regions is replaced with `id`. Looks like you have a mix of the two in the xml.